prawns/balances/src/handlers/typeGetters/getTreasuryEvents.ts

```diff
@@ -1,68 +1,107 @@
 import { EventHandlerContext } from '@subsquid/substrate-processor';
+import { Store } from '@subsquid/typeorm-store';
 import { SubstrateNetwork } from '../../model';
-import { TreasuryAwardedEvent as KhalaTreasuryAwardedEvent } from '../../types/khala/events';
-import { TreasuryAwardedEvent as LitentryTreasuryAwardedEvent } from '../../types/litentry/events';
-import { TreasuryAwardedEvent as LitmusTreasuryAwardedEvent } from '../../types/litmus/events';
+import { TreasuryAwardedEvent as KusamaTreasuryAwardedEvent } from '../../types/kusama/events';
+import { TreasuryAwardedEvent as PolkadotTreasuryAwardedEvent } from '../../types/polkadot/events';
+// import { TreasuryAwardedEvent as KhalaTreasuryAwardedEvent } from '../../types/khala/events';
```

Comment:
I guess these can be removed?

Reply:
ahhh Its because we don't yet have archives - ignore me

Reply:
I left these in intentionally so that it's quicker to add support when the new archives are ready (if I don't comment them my VS Code settings will remove them)